lib/core/src/lib/form/components/form-section/form-section.component.scss

```diff
@@ -1,10 +1,31 @@
+@use '../../../../../src/lib/styles/flex' as flex;
```

Comment:
I think this is incorrect import made by IDE?

Reply:
yes..